src/tribler-core/tribler_core/modules/metadata_store/restapi/tests/test_metadata_endpoint.py

```diff
@@ -228,7 +229,7 @@ def setUpPreSession(self):
             self.session.mds.TorrentMetadata(
                 infohash=infohash, title='ubuntu-torrent.iso', size=42, tracker_info=tracker_url, health=torrent_state
             )
-        url = 'metadata/torrents/%s/health?timeout=10&refresh=1' % hexlify(infohash)
+        url = 'metadata/torrents/%s/health?timeout=%s&refresh=1' % (hexlify(infohash), default_timeout)
```

Comment:
why not use f-string here? We're Python 3.7.

Reply:
yeah you can or you can not, but you should understand that few lines above there is not such  idiom as wel.

For this lines context i dont see such technical need.

https://github.com/Tribler/tribler/blob/fd39697c59b4a67f063591ed9d2621cc1b614d51/src/tribler-core/tribler_core/modules/metadata_store/restapi/tests/test_metadata_endpoint.py#L184

Reply:
This is not a strict requirement, but a good opportunity to modernise the code.

Reply:
it would be more ideal to have a separate work package or PR for that.